src/app/components/drawer/Projects/ProjectsComponent.js

```diff
@@ -71,15 +69,6 @@ const ProjectsComponent = ({
     setActiveItem({ type: listId, id: null });
   };
 
-  const handleClick = (route, id) => {
-    if (route !== activeItem.type || id !== activeItem.id) {
-      setActiveItem({ type: route, id });
-      if (collapsed) {
-        setCollapsed(false);
-      }
-    }
-  };
-
```

Comment:
As I said above, the `collapsed` state was not being used for anything. And the `setActiveItem` was redundant with a `useEffect` in that component.